src/app/modules/projects/components/project-card/project-card.component.scss

```diff
@@ -0,0 +1,152 @@
+@import './../../../../../assets/styles/theme';
+
+app-project-card {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    font-size: $font-size-m;
+    
+    .project-card-container {
+        display: flex;
+        flex-direction: row;
+        width: 90%;
+        margin-bottom: 3em;
```

Comment:
It's the wrapping container that should create gaps between each card, not the card itself.